Create About page for Email (#3053)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/about.md

---
nav_title: About Email
article_title: About Email
page_order: 0
description: "This reference article provides an overview of the Braze Email channel and common use cases."
channel:
  - email

---

# About email

> This article provides an overview of email at Braze, including the available customizations and potential use cases. In addition to the content covered here, we also recommend checking out our [Email](https://lab.braze.com/messaging-channels-email) LAB course.

> You can do a lot with email messages! Email messages are a great way to deliver rich content to your users. Email messaging has unlimited felxibility for communicating with your users outside of your app or website. Customize your email messages from managing your audience to including eye-catching multimmedia content. To see examples of email campaigns, check out our [Case Studies][6].

Reviewer: ```suggestion
You can do a lot with email messages! Email messages are a great way to deliver rich content to your users. Email messaging has unlimited flexibility for communicating with your users outside of your app or website. Customize your email messages from managing your audience to including eye-catching multimedia content. To see examples of email campaigns, check out our [Case Studies][6].
```

## Customize your emails

You can customize your email messaging in a variety of ways, including:

- [Braze email templates][2]
- [Custom HTML templates][7]
- [Drag & Drop Editor blocks][4]
- [User subscriptions][5]

## Great use cases

Here are some great use cases to get you inspired!

| Use case | Explanation |
| --- | --- |

> | Re-engagement | Reach users outside of your app, including those who have not installed the app |

Reviewer: ```suggestion
| Re-engagement | Reach users outside of your app, including those who have not installed the app. |
```

| Onboarding | Onboard and encourage new users to turn on push notifications or share the app on social networks. |

> | Rich messages | Allow for rich and dyanimic HTML messages

Reviewer: ```suggestion
| Rich messages | Allow for rich and dynamic HTML messages. |
```

> | Multimedia content | Ease of of multimedia content placement like videos and images that engage users. |

Reviewer: ```suggestion
| Multimedia content | Ease of multimedia content placement that engages users, like videos and images. |
```
